crates/walrus-sdk/src/config/communication_config.rs

```diff
@@ -270,4 +275,17 @@ pub(crate) mod default {
     pub fn max_data_in_flight() -> usize {
         12_500_000
     }
+
+    pub fn default_read_blob_certification_wait_time_secs() -> u64 {
+        {
+            #[cfg(test)]
+            {
+                30
+            }
+            #[cfg(not(test))]
+            {
+                0
+            }
+        }
```

Comment:
Important: I'm not sure this behaves as you expect it to behave: The `cfg(test)` only affects unit tests of the current crate, not tests in other crates or E2E tests AFAIU. A better solution would be to set the default value to `0` and then use a different value in the `default_for_test` function as mentioned above.